paimon-core/src/main/java/org/apache/paimon/operation/MergeFileSplitRead.java

```diff
@@ -338,6 +338,6 @@ private RecordReader<KeyValue> projectOuter(RecordReader<KeyValue> reader) {
     @Nullable
     public UserDefinedSeqComparator createUdsComparator() {
         return UserDefinedSeqComparator.create(
-                readerFactoryBuilder.readValueType(), sequenceFields);
+                readerFactoryBuilder.readValueType(), sequenceFields, true);
```

Comment:
Why here pass true?

Reply:
the same as upper. change it as options. @JingsongLi